test/LieTheory/WeylGroup.jl

```diff
@@ -187,9 +187,13 @@
   end
 
   b3_w0 = UInt8[1, 2, 1, 3, 2, 1, 3, 2, 3]
+  b3_s0 = [1=>1, 2=>1, 1=>1, 3=>1, 2=>1, 1=>1, 3=>1, 2=>1, 3=>1]
   b4_w0 = UInt8[1, 2, 1, 3, 2, 1, 4, 3, 2, 1, 4, 3, 2, 4, 3, 4]
+  b4_s0 = [1=>1, 2=>1, 1=>1, 3=>1, 2=>1, 1=>1, 4=>1, 3=>1, 2=>1, 1=>1, 4=>1, 3=>1, 2=>1, 4=>1, 3=>1, 4=>1]
   f4_w0 = UInt8[1, 2, 1, 3, 2, 1, 3, 2, 3, 4, 3, 2, 1, 3, 2, 3, 4, 3, 2, 1, 3, 2, 3, 4]
+  f4_s0 = [1=>1, 2=>1, 1=>1, 3=>1, 2=>1, 1=>1, 3=>1, 2=>1, 3=>1, 4=>1, 3=>1, 2=>1, 1=>1, 3=>1, 2=>1, 3=>1, 4=>1, 3=>1, 2=>1, 1=>1, 3=>1, 2=>1, 3=>1, 4=>1]
   g2_w0 = UInt8[1, 2, 1, 2, 1, 2]
+  g2_s0 = [1=>1, 2=>1, 1=>1, 2=>1, 1=>1, 2=>1]
```

Comment:
I don't like the duplication here, and the suffix `w0` was not referring to `word` in any way. But $w_0$ is how one usually writes the longest element of a weyl group in text books.
See my next comment for a workaround suggestion